### ISSUE TITLE
TypeScript types in React.FC wraps weirdly

### ISSUE THREAD

**Issue by SimenB** [prettier/prettier#7387]
**Prettier 1.19.1**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEBVADgEwEN4A1ASzgHclMAlOQsGAOgDEBhAHgB0pNMACmTABrTgCEArjBjQBAJwi40AGkzcQc+XDRoNmAD7qQAGwiF8ZKAHMNAPkwAyTFJnQA6mXwwAFgqV6UA4AvJgAFMCYWjqqmGYWVtZqaHAwrrJQnt4+mAC+AJSYwQ5hvPycdmX8mJwAImQAbl5w8pgA9JV81XWNza0dVZwDUPkA3LwgKpq4MGToyKCE8oqUAksIaMgghA0QXpMgAEbyjCKpAMq4jInIMPKScFM+MAC2Ju4+ZPBoV2Bw5xsvr0YABPLZgXQHKwpeQwBSEawvQjIABmhBMKSmACs0AAPcQnUQXQgvOAAGSscFR6MxIBxuPOiRMcAAipIIPBqRjHiArvIYVtQbgdGB5GQZgdcGLYFlfMgABwABimUogKXcJ1wWylOhaDSpUwAjuz4P5lFtCGgALRQOBwfD2g7aY1kbTwxHIpBo7lTFIvMi3e48tBM1kmqlemk8mCEQ6ynzIABMUzuhDIJkS7AgLyRWyg0ANIEkKQAKrHNpGfSAGg8AJJQB2wc6i8UwACCDfOoOZXJSuVyQA)
```sh
--parser typescript
```

**Input:**
```tsx
const UpdateView: React.FC<
  Pick<ButtonProps, "onPress" | "loading"> & ButtonWidthProps
> = ({ onPress, loading, setButtonWidth }) => (
  <>
    <Divider />
    <Divider />
  </>
);

```

**Output:**
```tsx
const UpdateView: React.FC<Pick<ButtonProps, "onPress" | "loading"> &
  ButtonWidthProps> = ({ onPress, loading, setButtonWidth }) => (
  <>
    <Divider />
    <Divider />
  </>
);

```

**Expected behavior:**
For it to match the input, which Flow, Babel and Babel-Flow parsers already do. It's just the TS parser

```tsx
const UpdateView: React.FC<
  Pick<ButtonProps, "onPress" | "loading"> & ButtonWidthProps
> = ({ onPress, loading, setButtonWidth }) => (
  <>
    <Divider />
    <Divider />
  </>
);
```

**Comment by thorn0** [comment]
This has been fixed by #6901, but not yet released.

**Comment by SimenB** [comment]
👍 

thoughts on adding "master" as a version to the playground?
